sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -205,7 +205,7 @@ void saveBulkEditWithFacilities(
 
 	List<CasePersonDto> getDuplicates(@Valid CasePersonDto casePerson);
 
-	List<CaseDataDto> getDuplicatesWithPathogenTest(@Valid CaseDataDto caseDataDto, PathogenTestDto pathogenTestDto);
+	List<CaseDataDto> getDuplicatesWithPathogenTest(@Valid PersonReferenceDto casePersonReferenceDto, PathogenTestDto pathogenTestDto);
```

Comment:
rename `casePersonReferenceDto` to `personReferenceDto` as this method has been extended to handle persons from other entities, too.